Sources/URLRouting/PathEnd.swift

```diff
@@ -1,16 +0,0 @@
-public struct PathEnd: ParserPrinter {
```

Comment:
Made this internal to `Route` since it also isn't intended to be used directly. This also simplifies public types/docs.